MakeFile

```
@@ -0,0 +1,50 @@
+.PHONY: db db-stop db-reset migrate migrate-revert db-logs migration-new
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)